packages/react-scripts/template/styleguide.config.js

```diff
@@ -1,6 +1,6 @@
 require('dotenv')
 const styles = require('@fs/styleguidist-overrides/dist/styles.js')
-const webpackConfig = require('@fs/configurations/webpack.dev.js')
+const webpackConfig = require('@fs/react-scripts/config/webpack.config.js')
```

Comment:
there are a few things going on in @fs/configuration's webpack.config.js file that aren't in react-scripts webpack.config.js.

Do we need any of them? Like the setupProxy stuff per chance?

Reply:
No, this should be fine.

Reply:
`setupProxy` is pulled in at runtime. If we didn't have a styleguidist config, it would be pulling in this webpack directly.